firebase/src/core/domain/models/memo.ts

```diff
@@ -1,16 +1,36 @@
 import { defaultMaxStringLength, validate } from "#utils/validate";
 import * as Joi from "joi";
 
+interface MemoContent {
+  insert: string;
+  attributes?: {
+    bold?: boolean;
+    italic?: boolean;
+    underline?: boolean;
+    codeBlock?: boolean;
+  };
+}
+
 export interface Memo {
-  readonly id: string;
-  readonly question: Record<string, unknown>[];
-  readonly answer: Record<string, unknown>[];
+  readonly uniqueId: string;
```

Comment:
Just to point that I personally do prefer `id` over `uniqueId`, as I mentioned when we were developing Flutter client, but we need to create a standard. Or everything becomes `id` or `uniqueId`. We already have `uniqueId` set as the property in the flutter models and collections json's

Reply:
But should we update the collections jsons and also Flutter model to use id as well? Otherwise we'll use `id` in the server and `uniqueId` in the client. I highlight what I've said back in the client development that `id` implicitly means that's an unique identifier IMO, that's no reason to emphasis the `unique` stuff.

PS: We would need this minor refactoring in a separate PR instead mixing with this one stuff